src/navigation/MainTabNavigator.tsx

```diff
@@ -1,37 +1,154 @@
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-import { colors, device, normalize } from '../config';
-import { TabNavigatorConfig } from '../types';
+import { LoadingSpinner } from '../components';
+import { colors, normalize } from '../config';
+import {
+  createDynamicTabConfig,
+  tabNavigatorConfig as defaultTabNavigatorConfig
+} from '../config/navigation/tabConfig';
+import { useStaticContent } from '../hooks';
+import { ScreenName, TabConfig, TabNavigatorConfig } from '../types';
 
 import { getStackNavigator } from './AppStackNavigator';
 
+const defaultTabRoutes: TabRoutes = {
+  tabConfig: {
+    activeBackgroundColor: colors.surface,
+    activeTintColor: colors.accent,
+    inactiveBackgroundColor: colors.surface,
+    inactiveTintColor: colors.primary,
+    tabs: ['Home', 'Service', 'Events', 'Index', 'About']
```

Comment:
why is `Index` a default? i suggest a default of the ones we have in `tabNavigatorConfig`, so:

```
homeTabConfig,
serviceTabConfig,
pointOfInterestTabConfig,
eventsTabConfig,
aboutTabConfig
```